sdk_extensions/logging/support/src/main/java/io/opentelemetry/logging/api/Exporter.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.logging.api;
+
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import java.util.Collection;
+
+/**
+ * An exporter is responsible for taking a list of {@link LogRecord}s and transmitting them to their
+ * ultimate destination.
+ */
+public interface Exporter {
+  CompletableResultCode accept(Collection<LogRecord> records);
+
+  void shutdown();
```

Comment:
SpanExporter.shutdown returns CompletableResultCode. Should we too for consistency? Generally I think it is worth trying to be consistent with our tracing peers such as SpanExporter.